pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -103,11 +103,12 @@ Listen Sockets
                                   higher than 0 to enable TCP Fast Open when available.
                                   Default is 0.
 
-.. function:: addDOHLocal(address, certFile(s), keyFile(s) [, urls [, options]])
+.. function:: addDOHLocal(address, [certFile(s) [, keyFile(s) [, urls [, options]]]])
```

Comment:
I think it should be 

```
function:: addDOHLocal(address, [certFile(s) , keyFile(s) [, urls [, options]]])
```

Since if you have `certFile` then you should have `keyFile` too.

Reply:
In practice that's pretty much what the code does, it requires the same number of keys.